core/handlers.py

```diff
@@ -107,6 +107,44 @@
     )
 
 
+@dp.message_handler(commands=["peek"], state="*")
```

Comment:
The `peek` name isn't self-descriptive nor intuitive, let's call it `next_reminder`
```suggestion
@dp.message_handler(commands=["next_reminder"], state="*")
```